client/packages/electron/src/electron.ts

```diff
@@ -25,28 +28,42 @@ if (require('electron-squirrel-startup')) {
   app.quit();
 }
 
+const connectToServer = (window: BrowserWindow, server: FrontEndHost) => {
+  isDiscovering = false;
+  const url = frontEndHostUrl(server);
+  // Can change 'frontEndHostUrl(server)' to http://localhost:3003 if you want client to display front end served by webpack
+  window.loadURL(url);
+  connectedServer = server;
+  systemPreferences.setUserDefault('/mru/server', 'string', url);
+};
+
 const start = (): void => {
   // Create the browser window.
   const window = new BrowserWindow({
-    height: 600,
-    width: 800,
+    height: 768,
```

Comment:
the default is 800x600 if not specified. and that's fine.. but the login screen shows slightly better if slightly wider, so here we go.